src/Builder/Visitor/AddClassToClassListVisitor.php

```diff
@@ -225,7 +225,11 @@ protected function addArrayElementToPosition(Node $node): array
         foreach ($node->items as $item) {
             $nodeValue = $item->value->name->toString();
             if (!($item->value instanceof MethodCall)) {
-                $nodeValue = sprintf('%s::%s', $item->value->class->toString(), $item->value->name->toString());
+                $nodeValue = sprintf(
+                    '%s::%s',
+                    (new ClassHelper())->getShortClassName((string)$item->value->class->toString()),
```

Comment:
Why do you needs short class here?

Reply:
Because the strategy store Before and after keys without a namespace 
Created https://spryker.atlassian.net/browse/SDK-4580

Reply:
Sounds like a bug. All classes must have full namespace in manifests.